homeassistant/components/broadlink/device.py

```diff
@@ -24,41 +31,42 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def get_domains(device_type):
+def get_domains(device_type: str) -> set[Platform]:
     """Return the domains available for a device type."""
     return {d for d, t in DOMAINS_AND_TYPES.items() if device_type in t}
 
 
 class BroadlinkDevice:
     """Manages a Broadlink device."""
 
-    def __init__(self, hass, config):
+    api: blk.Device
+
+    def __init__(self, hass: HomeAssistant, config: ConfigEntry) -> None:
         """Initialize the device."""
         self.hass = hass
         self.config = config
-        self.api = None
         self.update_manager = None
-        self.fw_version = None
-        self.authorized = None
-        self.reset_jobs = []
+        self.fw_version: int | None = None
```

Comment:
Just for reference, this is defined here:
https://github.com/mjg59/python-broadlink/blob/3c183eaaef6cbaf9c1154b232116bc130cd2113f/broadlink/device.py#L242